Zend/tests/bug70630.phpt

```diff
@@ -0,0 +1,23 @@
+--TEST--
+Bug #70630 (Closure::call/bind() crash with ReflectionFunction->getClosure())
+--FILE--
+<?php
+
+class a {}
+function foo() { print "ok\n"; }
+
+foreach (["substr", "foo"] as $fn) {
+	$x = (new ReflectionFunction($fn))->getClosure();
+	$x->call(new a);
+	Closure::bind($x, new a, "a");
+}
+
+?>
+--EXPECTF--
+
+Warning: substr() expects at least 2 parameters, 0 given in %s on line %d
+
+Warning: Cannot bind function substr to a class scope in %s on line %d
```

Comment:
Ehm… I'm not allowing scope binding to a, I'm allowing `Closure::bind($x, new a)` standalone though.